modules/compliance-new-tailored-profiles.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-tailor.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-new-tailored-profiles_{context}"]
+= Creating a new tailored profile
+
+You can write a tailored profile from scratch using the TailoredProfile object. Set an appropriate `title` and `description` and leave the `extends` field empty. Indicate to the Compliance Operator what type of scan will this custom profile generate:
+
+* Node scan: Scans the Operating System.
+* Platform scan: Scans the OpenShift configuration.
+
+Set the following annotation on the TailoredProfile object:
+
+.Example `new-profile.yaml`
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: new-profile
+spec:
+  extends: <1>
+  description: My custom profile <2>
+  title: Custom profile <3>
+compliance.openshift.io/product-type: Node/Platform <4>
```

Comment:
It is a good example. It is not the right format for the product-type. Should be included in the annotations.
The product-type could be Platform or Node. 
Besides, since extends is not mandatory, add enableRules here maybe more reasonable. 

```
apiVersion: compliance.openshift.io/v1alpha1
kind: TailoredProfile
metadata:
  name: new-profile
  annotations:
    compliance.openshift.io/product-type: Node
spec:
  extends: 
  description: My custom profile 
  title: Custom profile 
```
Better to use below example:
```
apiVersion: compliance.openshift.io/v1alpha1
kind: TailoredProfile
metadata:
  name: testprofile
  namespace: openshift-compliance
  annotations:
    compliance.openshift.io/product-type: Node
spec:
  extends: rhcos4-moderate
  description: test
  title: test
  enableRules:
    - name: rhcos4-audit-rules-dac-modification-chmod
      rationale: test
```


Details seen from this: http://pastebin.test.redhat.com/1078351